dask_jobqueue/pbs.py

```diff
@@ -83,15 +83,18 @@ def __init__(self, queue=None, project=None, resource_spec=None, walltime=None,
             header_lines.append('#PBS -q %s' % queue)
         if project is not None:
             header_lines.append('#PBS -A %s' % project)
-        if resource_spec is None:
+        if resource_spec is None and self.worker_threads:
             # Compute default resources specifications
             ncpus = self.worker_processes * self.worker_threads
```

Comment:
Should'nt we store cores and memory in jobqueue abstract class. So that we don't have to perform the inverse computation as in https://github.com/dask/dask-jobqueue/pull/86/files#diff-12a0c7384462f5473600598f508cac74R161 ?

I think PBS or Slurm part only need the cores and memory part.

Reply:
Can do

Reply:
Done